test-app/tests/integration/components/toucan-form/form-radio-group-test.gts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import { render } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+import ToucanForm from '@crowdstrike/ember-toucan-form/components/toucan-form';
+
+interface TestData {
+  radio?: string;
+}
+
+module('Integration | Component | ToucanForm | RadioGroup', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it sets the readonly attribute with `@isReadOnly` at the root', async function (assert) {
```

Comment:
Radio test is already over here: https://github.com/CrowdStrike/ember-toucan-core/blob/main/test-app/tests/integration/components/radio-test.gts#L45

Reply:
Yeah! So this is where things get a little tricky I guess.  Since this is in `<ToucanForm`, I want to ensure that the component argument gets passed from `ToucanForm` => Core's `RadioGroup` properly.  That way we can ensure the component arguments are being mapped properly since we don't have splat-arguments yet